### ISSUE TITLE
Build guide steel support wire details missing

### ISSUE THREAD

**Issue by zedn4c3r** [bigtreetech/EBB#25]
The build guide shows a steel wire is used to support the umbilical cable, and gives great details on how to bend it, but fails to say what size or type of steel wire to use.

**Comment by dazkit** [comment]
I modified a remixed mod from umbilical that uses a 0.8mm Steel wire. You can download it here: https://www.printables.com/model/383128-voron-24r2-pg7-cable-gland-with-spring-steel-wire-

![NEU](https://user-images.githubusercontent.com/45803283/215348376-906b7378-e701-41cd-bc1c-6369d97b5ca7.png)

**Comment by jcfain** [comment]
> The build guide shows a steel wire is used to support the umbilical cable, and gives great details on how to bend it, but fails to say what size or type of steel wire to use.

Yes im trying to figure out what gauge I should buy right now...

**Comment by dazkit** [comment]
Piano wire with a diameter of 0.81 is recommended from many discord users
